src/pages/LandingPage/LandingPage.jsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react'; // <-- useState is a react hook and imported here, you must always import react hooks
 import scrollToElement from '../../shared/utility/scrollToElement';
 import PortfolioCard from './PortfolioCard';
 import ContactForm from './ContactForm';
 import Render3DShape from './Render3DShape';
 
+const POSSIBLE_SHAPES = [
+  'torus',
+  'sphere',
+  'dodecahedron',
+  'icosahedron',
+  'octahedron',
+  'tetrahedron',
+  'torusknot',
+  'sahilsknot'
+];
+
+const INITIAL_SHAPE_INDEX = 0;
+const INITIAL_SHAPE = POSSIBLE_SHAPES[INITIAL_SHAPE_INDEX];
+
 const LandingPage = () => {
   const heroImage = 'hero.jpg';
+  const [shape, setShape] = useState(INITIAL_SHAPE);
+  let prevShapeIndex = INITIAL_SHAPE_INDEX;
 
```

Comment:
Here i use the INITIAL_SHAPE as the default for the useState hook, so that whenever the landingpage re-renders it gets the torus shape.